tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -3,6 +3,7 @@ import path from 'path';
 
 import yazl from 'yazl';
 import yauzl from 'yauzl';
+import { dirSync } from 'tmp-promise';
```

Comment:
Nit, would you mind to import `"tmp-promise"` as `tmp` and then use `dirSync` as `tmp.dirSync`?
we are going to use it in a single place (the `prepareTmpDir`) and using it `tmp.dirSync` would make it immediately clear (at the cost of just 4 more characters).

Reply:
Done!

Reply:
@marchiore I think that I may have commented at the same time when you where writing this comment :-D
and so I guess that you may have missed the new one here https://github.com/mozilla/addons-linter/pull/3095#pullrequestreview-390009513 ;-)